### PR TITLE
Solr suggester

### LINKED ISSUE #43: Suggester files can fill up the container
By default jetty uses as tmp folder /tmp
In case suggester is being used, it can create big files which might fill up the container rootfs.

Possible workaround: set the jetty tmp to a folder which is declared as a volume:

JAVA_OPTS_JETTYTMP=-Djava.io.tmpdir=/opt/alfresco-search-services/temp

### PR DESCRIPTION
**Pull request by anghelutar**
Fixes #43

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -24,9 +24,11 @@ def includeSubprojectPerVersion(flavor,classification) {
                 def version = projectFile.name
                 def projectName = "upstream:${flavor}-${version}"
 
-                include projectName
-                project(":${projectName}").projectDir = file("upstream/${flavor}/${version}")
-                println "included $projectName"
+                if (new File("$projectFile/overload.gradle").exists()) {
```
Comment (vierbergenlars) [inline settings.gradle:27]: Why do we need to check if this overload exists everywhere?

Comment (anghelutar) [reply settings.gradle:27]: This is there to catch directories that are left behind when a subproject is renamed.

Comment (vierbergenlars) [reply settings.gradle:27]: So same comment as on https://github.com/xenit-eu/docker-alfresco/pull/69: please add a comment to the code.

Comment (anghelutar) [reply settings.gradle:27]: Would have been faster if you added it yourself.

**Review by RVanhuysseXenit — APPROVED** [review]

**Pull request merged**